{
  "schema_version": "1.2.0",
  "id": "GHSA-h3qr-fjhm-jphw",
  "modified": "2022-07-15T18:40:09Z",
  "published": "2022-07-14T00:00:23Z",
  "aliases": [
    "CVE-2019-10800"
  ],
  "summary": "Codecov prior to 2.0.16 does not sanitize gcov arguments",
  "details": "This affects the package codecov before 2.0.16. The vulnerability occurs due to not sanitizing gcov arguments before being being provided to the popen method.",
  "severity": [

  ],
  "affected": [
    {
      "package": {
        "ecosystem": "PyPI",
        "name": "codecov"
      },
      "ranges": [
        {
          "type": "ECOSYSTEM",
          "events": [
            {
              "introduced": "0"
            },
            {
              "fixed": "2.0.16"
            }
          ]
        }
      ]
    }
  ],
  "references": [
    {
      "type": "ADVISORY",
      "url": "https://nvd.nist.gov/vuln/detail/CVE-2019-10800"
    },
    {
      "type": "WEB",
      "url": "https://github.com/codecov/codecov-python/commit/2a80aa434f74feb31242b6f213b75ce63ae97902"
    },
    {
      "type": "WEB",
      "url": "https://snyk.io/vuln/SNYK-PYTHON-CODECOV-552149"
    },
    {
      "type": "PACKAGE",
      "url": "https://github.com/codecov/codecov-python"
    }
  ],
  "database_specific": {
    "cwe_ids": [

    ],
    "severity": "MODERATE",
    "github_reviewed": true
  }
}